app/src/main/java/org/sopt/sample/presentation/common/base/BindingFragment.kt

```diff
@@ -0,0 +1,32 @@
+package org.sopt.sample.presentation.common.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.fragment.app.Fragment
+
+abstract class BindingFragment<T : ViewDataBinding>(@LayoutRes val layoutRes: Int) : Fragment() {
+    private var _binding: T? = null
+    val binding: T
+        get() = requireNotNull(_binding)
```

Comment:
```suggestion
    private var _binding: T? = null
    val binding: T
        get() = requireNotNull(_binding) { "${this::class.java.simpleName} error." }
```

예외 처리를 추가해줘도 좋을 것 같습니다!

Reply:
오 좋아요~~